packages/model/src/research-output.ts

```diff
@@ -6,7 +8,7 @@ export type ResearchOutputAuthor =
   | { readonly id: string };
 
 export type ResearchOutputFormData = {
-  readonly url: string;
+  readonly url?: string;
```

Comment:
Why this change? In the form design it was a mandatory field

Reply:
The proposal doesn't have a URL. But it's also not created by users.
Is keeping the URL required for the form and change it on the ResearchOutputResponse an option?

Reply:
Yeah so this type was specifically for the input form data, I don't think it even reaches the backend before it is re-mapped